diff --git a/rtl/sdram.sv b/rtl/sdram.sv
index 8ba42ed..1442950 100644
--- a/rtl/sdram.sv
+++ b/rtl/sdram.sv
@@ -71,10 +71,10 @@ assign SDRAM_nCS = 0;
 assign SDRAM_CKE = 1;
 assign {SDRAM_DQMH,SDRAM_DQML} = SDRAM_A[12:11];
 
-localparam RASCAS_DELAY   = 3'd2; // tRCD=20ns -> 2 cycles@85MHz
+localparam RASCAS_DELAY   = 3'd3; // Pocket: tRCD=20ns -> 3 cycles@107MHz (as openFPGA-Genesis)
 localparam BURST_LENGTH   = 3'd0; // 0=1, 1=2, 2=4, 3=8, 7=full page
 localparam ACCESS_TYPE    = 1'd0; // 0=sequential, 1=interleaved
-localparam CAS_LATENCY    = 3'd2; // 2/3 allowed
+localparam CAS_LATENCY    = 3'd3; // Pocket: CL3 at 107MHz (as openFPGA-Genesis)
 localparam OP_MODE        = 2'd0; // only 0 (standard operation) allowed
 localparam NO_WRITE_BURST = 1'd1; // 0=write burst enabled, 1=only single access write
 
