diff --git a/rtl/32X/32X.sv b/rtl/32X/32X.sv
index 2709f85..90616f6 100644
--- a/rtl/32X/32X.sv
+++ b/rtl/32X/32X.sv
@@ -140,7 +140,7 @@ module S32X
 	bit         TXDS;
 	bit         SCKM;
 	bit         SCKS;
-	SH7604 #(.BUS_AREA_TIMIMG(4'b1000), .BUS_SIZE_BYTE_DISABLE(1)) MSH
+	SH7604 #(.BUS_AREA_TIMIMG(4'b1000), .BUS_SIZE_BYTE_DISABLE(1), .UBC_DISABLE(1)) MSH
 	(
 		.CLK(CLK),
 		.RST_N(RST_N),
@@ -196,7 +196,7 @@ module S32X
 		.FAST(0)
 	);
 	
-	SH7604 #(.BUS_AREA_TIMIMG(4'b1000), .BUS_SIZE_BYTE_DISABLE(1)) SSH
+	SH7604 #(.BUS_AREA_TIMIMG(4'b1000), .BUS_SIZE_BYTE_DISABLE(1), .UBC_DISABLE(1)) SSH
 	(
 		.CLK(CLK),
 		.RST_N(RST_N),
